recipe/meta.yaml

```diff
@@ -1,29 +1,33 @@
+{% set name = "jupyter_bokeh" %}
 {% set version = "3.0.7" %}
 
 package:
-  name: jupyter_bokeh
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyter_bokeh/jupyter_bokeh-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 2da8c3ddc734d15737bf06126d9e31e84d30f18ac3da3a3f95be40a95a054c87
 
 build:
-  number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --install-option="--skip-npm"
+  number: 1
+  skip: True # [s390x or  py<38]
+  script: {{ PYTHON }} -m pip install . -vv --install-option="--skip-npm" --no-deps
 
 requirements:
+  build:
+    - {{ compiler('c') }}
   host:
-    - jupyter-packaging >=0.7.9,<0.8
+    - jupyter-packaging 0.7.9
     - pip
-    - python >=3.8
+    - python
+    - setuptools
+    - wheel
+    - jupyterlab >=3,<4
```

Comment:
I think we should use exact pinning in the host section. In this case jupyterlab 3.5.3 as we are building it for py311 too, I think, aren't we?